### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by DilsherB**
in this PR:
1. Following Components Created.
    a. Books
    b. Categories
    c. BookDetail
    d. CompletionCircle
    e. Menu
    f. NewBookForm
2. React-router-dom installed.
3. Basic setup of routers done.
4. No linter errors
5. git flow followed.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @DilsherB, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the README file that does not use the Microverse template. Please use the [Microverse template](https://github.com/microverseinc/readme-template).

- [ ] In the case of README consider not leaving out any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.

| professional Readme  | Your Readme that is not professional |
| ------------- | ------------- |
|  Recording 2023-02-14 at 12 52 20 (5)	  | image |

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import BookProgress from "./BookProgress";
+
+const Books = () => {
+  return (
+    <div>
+      <BookDetail />
+      <BookAction />
+      <BookProgress />
+    </div>
+  );
```
Comment (uwadonat) [inline src/components/Books.js]: Hello @DilsherB 
- [ ] As requested in the project requirements, let make sure below features are implemented:
- Contains individual book state, as well as a button to delete the book (the button wouldn't remove the books *yet*, this is just a placeholder for now)
- Contains a list to renders individual books
- Contains a form to create new books (with inputs for the title, the author of the book and a button to submit)
   the button wouldn't add the book *yet*, this is just a placeholder for now

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @DilsherB,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Detailed pull request message ✔️
- Professional readme files ✔️
- Correct workflow was used ✔️
almost

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @DilsherB 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**